app/services/analytics_events.rb

```diff
@@ -0,0 +1,55 @@
+module AnalyticsEvents
+  # @identity.idp.event_name Account Reset
```

Comment:
👀 custom event name! we can make a form of the script that errors when this is absent as a lint